src/interface/news/ImageArticle.tsx

```diff
@@ -1,7 +1,13 @@
+/* eslint react/prop-types: 0 */
 import React from 'react';
-import PropTypes from 'prop-types';
 
-const ImageArticle = props => {
+interface Props {
+  title: React.ReactNode,
+  image: React.ReactNode,
+  style: object
+}
+
+const ImageArticle: React.FC<Props> = (props) => {
```

Comment:
```suggestion
const ImageArticle = (props: Props) => {
```

I prefer this style since it doesn't add an incorrect children prop and doesn't rely on type conversion; it just is a component given that Props is of the correct type.